unified-codes/apps/web/src/components/explorer/ExplorerTableRows.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
The styles are missing - coming soon?
here's what I implemented to make it like it was:

```
const styles = (theme: ITheme) => {
  const row = {
    cursor: 'pointer',
    '&:hover': { backgroundColor: theme.palette.background.toolbar },
    '& td:last-child': { borderRight: 0 },
    '& td:first-child': { fontWeight: 700 },
  };
  return {
    root: {},
    rowPrimary: {
      ...row,
      backgroundColor: theme.palette.background.highlight,
    },
    rowSecondary: row,
    cell: {
      borderRight: `1px solid ${theme.palette.divider}`,
      borderBottom: 0,
    },
  };
};
```

Reply:
Can you remember what colour `highlight` was? I can't find it!

Reply:
dc05abe74f84b462acff354536fcbdb33bbdd4e4 (colours may need some tweaking!).

Reply:
the highlight was a grey (#f5f5f5 I think) but it looks better as the toolbar background colour, I think. So I used that above